native/mod_manager/mod_manager.c

```diff
@@ -1206,6 +1206,36 @@ static void mark_node_removed(nodeinfo_t *node)
         node->mess.num_remove_check = 0;
     }
 }
+static proxy_worker_shared *read_shared_by_node(request_rec *r, nodeinfo_t *node)
```

Comment:
Thinking about this returning pointer to the shared memory, I would suggest adding `const`.

Reply:
fixed.